actionview/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Add the template digest to the cache key for collection caching.
```

Comment:
I just tried with [this sample app](https://github.com/repinel/my_app/tree/test-collection-cache) and the digest is not being attached to the cache key.

Using Rails from `master`:
```
cache/A87/A60:
views%2Fusers%2F2-20150620185152601259000

cache/A9D/470:
views%2Fusers%2F1-20150620185147798585000
```

With this fix:
```
cache/4DB/781:
views%2Fusers%2F2-20150620185152601259000%2F65ffadfc89bbc5250bc31f768b90bfdc

cache/4F1/1B1:
views%2Fusers%2F1-20150620185147798585000%2F65ffadfc89bbc5250bc31f768b90bfdc
```

@kaspth What do you think?